examples/rsp-next-ts/pages/index.tsx

```diff
@@ -0,0 +1,394 @@
+import Head from "next/head";
+import styles from "../styles/Home.module.css";
+import React from "react";
+import {
+  ActionMenu,
+  Item,
+  Menu,
+  MenuTrigger,
+  ListView,
+  Calendar,
+  DateField,
+  DatePicker,
+  DateRangePicker,
+  RangeCalendar,
+  TimeField,
+  Badge,
+  LabeledValue,
+  Picker,
+  ActionButton,
+  Breadcrumbs,
+  Button,
+  ButtonGroup,
+  Cell,
+  Checkbox,
+  CheckboxGroup,
+  Column,
+  Divider,
+  Flex,
+  Form,
+  Grid,
+  Heading,
+  Image,
+  Link,
+  ListBox,
+  LogicButton,
+  NumberField,
+  Radio,
+  RadioGroup,
+  Row,
+  SearchField,
+  Switch,
+  TableBody,
+  TableHeader,
+  TableView,
+  TabList,
+  TabPanels,
+  Tabs,
+  Text,
+  TextArea,
+  TextField,
+  ToggleButton,
+  DialogTrigger,
+  AlertDialog,
+  ContextualHelp,
+  Content,
+  Dialog,
+  Header,
+  TooltipTrigger,
+  Tooltip,
+  ComboBox,
+  RangeSlider,
+  Slider,
+  Meter,
+  ProgressBar,
+  ProgressCircle,
+  StatusLight,
+  Footer,
+  IllustratedMessage,
+  Keyboard,
+  View,
+  Well,
+} from "@adobe/react-spectrum";
+import Edit from "@spectrum-icons/workflow/Edit";
+import NotFound from "@spectrum-icons/illustrations/NotFound";
+import Section from "../components/Section";
+
+export default function Home() {
+  return (
+    <div className={styles.container}>
+      <Head>
+        <title>React Spectrum + NextJS</title>
+        <meta name="description" content="React Spectrum + NextJS" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+
+      <main className={styles.main}>
+        <Heading level={1}>
+          React Spectrum +{" "}
+          <Link>
+            <a href="https://nextjs.org">Next.js</a>
+          </Link>
+        </Heading>
+
+        <Flex direction="column" gap="size-125">
```

Comment:
With Next.js are we just confirming that they load or should we spread around some props (quiet, emphasized, etc.) throughout the file?

Reply:
IMO any failures would likely happen on initial load or interactivity, so I think it's fine not to add every style variant.